### PR TITLE
Added note about full-consensus-node storage requirements

### LINKED ISSUE #648: Add note about full-consensus-node storage requirements
https://github.com/celestiaorg/docs/pull/641#discussion_r1164568713

### LINKED ISSUE #802: clarify that an archive node is a consensus full node with no pruning

### PR DESCRIPTION
**Pull request by AnmolSirola**
### Overview
Resolves #648 
Resolves #802 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/nodes/full-consensus-node.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Full Consensus Nodes:
 * Disk: **250 GB SSD Storage**
 * Bandwidth: **1 Gbps for Download/1 Gbps for Upload**
 
+**Note**: Running a full consensus node requires significant storage capacity to store the entire blockchain history. As of the latest recommendation, it is advisable to have at least 250 GB of SSD storage for a Celestia full consensus node. Please ensure that your storage meets this requirement to ensure smooth syncing and operation of the node.
```
Comment (rootulp) [inline docs/nodes/full-consensus-node.md:25]: [not blocking] IMO this seems redundant with the line above.

> * Disk: **250 GB SSD Storage**

so I don't think the addition in this PR is necessary but defer to docs owners

Comment (jcstein) [reply docs/nodes/full-consensus-node.md:25]: good call. apologies for the confusion here. i believe that the original discussion resulted in:
- recommended storage: 250gb
- recommended storage without pruning (which would live in the "note": 500gb, context [here](https://github.com/celestiaorg/docs/pull/641#discussion_r1164568713)

Comment (jcstein) [reply docs/nodes/full-consensus-node.md:25]: so the note should include the recommendation to use more storage if not using pruning, thank you for the PR @AnmolSirola 🙌

Comment (jcstein) [inline docs/nodes/full-consensus-node.md:25]:
```suggestion
**Note**: Running a full consensus node requires significant storage capacity to store the entire blockchain history. As of the latest recommendation, it is advisable to have at least 250 GB of SSD storage for a Celestia full consensus node if you are using pruning. If you are not using pruning, it is recommended to have 500 GB of SSD storage. Please ensure that your storage meets this requirement to ensure smooth syncing and operation of the node.
```

Comment (AnmolSirola) [reply docs/nodes/full-consensus-node.md]: @jcstein Made the required changes :)

**Review by jcstein — APPROVED** [review]

File: docs/nodes/full-consensus-node.md (hunk 1 above)
Comment (jcstein) [reply docs/nodes/full-consensus-node.md]: thank you for the contribution @AnmolSirola !

File: docs/nodes/full-consensus-node.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ Full Consensus Nodes:
 * Disk: **250 GB SSD Storage**
 * Bandwidth: **1 Gbps for Download/1 Gbps for Upload**
 
+**Note**: Running a full consensus node requires significant storage capacity to store the entire blockchain history. As of the latest recommendation, it is advisable to have at least 250 GB of SSD storage for a Celestia full consensus node if you are using pruning. If you are not using pruning, it is recommended to have 500 GB of SSD storage. Please ensure that your storage meets this requirement to ensure smooth syncing and operation of the node.
```
Comment (jcstein) [inline docs/nodes/full-consensus-node.md:25]:
```suggestion
**Note**: Running a full consensus node requires significant storage capacity to store the entire blockchain history. As of the latest recommendation, it is advisable to have at least 250 GB of SSD storage for a Celestia full consensus node if you are using pruning. If you are not using pruning, you are running an archive node, and it is recommended to have 500 GB of SSD storage. Please ensure that your storage meets this requirement to ensure smooth syncing and operation of the node.
```

**Pull request merged**